src/style.css

```diff
@@ -1 +1,86 @@
 /* Styles go here */
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.logo {
+  display: flex;
+  align-self: center;
```

Comment:
`align-self` is used to overwrite the `align-items` of the parent for each individual item. the parent here is `.header` and its `align-items' prop is set to `center`

Reply:
Okay, I didn't know this property gets inherited.